packages/dx-react-grid-bootstrap3/src/templates/group-panel-cell.jsx

```diff
@@ -18,7 +18,7 @@ export const GroupPanelCell = ({
     <span
       onClick={(e) => {
         if (!allowSorting) return;
-        changeSortingDirection({ keepOther: e.shiftKey });
+        changeSortingDirection({ keepOther: e.shiftKey || e.ctrlKey, remove: e.ctrlKey });
```

Comment:
`Ctrl + Click` is used in Windows and Linux. On macOS, the following keyboard shortcut is used: `CMD + Click`